src/pages/advertiser/screens/AdvertiserAdvertsTable/AdvertiserAdvertsTable.tsx

```diff
@@ -100,7 +100,7 @@ const AdvertiserAdvertsTable = ({ advertiserId }: TAdvertiserAdvertsTableProps)
                 <Tab className='text-xs' title={localize('Buy')} />
                 <Tab title={localize('Sell')} />
             </Tabs>
-            <AdvertsTableRenderer data={data} isLoading={isLoading} loadMoreAdverts={loadMoreAdverts} />
+            <AdvertsTableRenderer data={data} isLoading={isLoading || isRefetching} loadMoreAdverts={loadMoreAdverts} />
```

Comment:
Added this here because after blocking/unblocking the user, the advertisers adverts disappear without a loader shown. So it looks a little glitchy.

Before:

https://github.com/user-attachments/assets/661d4fc9-1f15-4136-b587-5011735eb298



After

https://github.com/user-attachments/assets/588fa8b3-3bd5-4290-afb0-5245096d7f25